bin/generateLocaleManifest.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+
+##
+# This script is designed to generate the chrome.manifest for the add-on in
+# the propert format, taking into account the locales present in the repo.
+##
+
+from __future__ import print_function
+
+import argparse
+import io
+import os
+import re
+import localeUtils
+
+# defaults
+DEF_L10N_SRC = os.path.join("locale")
+DEF_INPUT_FILE_NAME = os.path.join("locale", os.extsep.join(["jar", "mn"]))
+DEF_OUTPUT_FILE_NAME = os.path.join("built", "add-on", "chrome", "locale",
+                                    os.extsep.join(["chrome", "manifest"]))
+
+
+def main(l10n_src, input_file_name, output_file_name):
+    print("Using l10n tree from", l10n_src)
```

Comment:
We can drop the `print` and `import` (also in `generateChromeManifest`) as these aren't that useful.